common_test.go

```diff
@@ -182,6 +182,12 @@ func assertTrue(t *testing.T, description string, value bool) {
 	}
 }
 
+func assertFalse(t *testing.T, description string, value bool) {
```

Comment:
This isn't necessary and hides the location of the error in the tests, unless there is a lot of repetition to warrant a function

Reply:
It's just a simple counterpart for assertFalse for asserting a false value. Of course, it's possible to use `assert.Equal(t, description, value, false)` instead, but I feel that assertFalse() better expresses the intent.

I can remove it of course.